### PR TITLE
chore(php): Add two new libraries into the php image

### PR DESCRIPTION
**Pull request by c-Rolland**
### **Subject**
 I m targeting this branch to add the following libraries into the php image: `libcrypto1.1` `libssl1.1`.

### CODE REVIEW

File: php/Dockerfile.j2 [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN echo "Starting ..." && \
     echo "@edge-community http://nl.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
     apk add --update --upgrade alpine-sdk apk-tools@edge-main autoconf bash bzip2 cyrus-sasl-dev curl freetype-dev gettext git \
         gnu-libiconv@edge-community icu-dev@edge-main libjpeg-turbo-dev libmcrypt-dev libmemcached-dev libpng-dev libssh2-dev \
-        libxml2-dev libzip-dev make mysql-client openssh-client patch postgresql-dev rsync tzdata && \
+        libxml2-dev libzip-dev make mysql-client openssh-client patch postgresql-dev rsync tzdata libcrypto1.1 libssl1.1 && \
```
Comment (mremi) [inline php/Dockerfile.j2:28]: Please sort by name

Comment (c-Rolland) [reply php/Dockerfile.j2]: done

Comment (c-Rolland) [reply php/Dockerfile.j2]: I have no idea why the CI fail...  I was build and test the php image in local successfully. :/

**Comment by c-Rolland** [comment]
I have no idea why the CI fail... I was build and test the php image in local successfully. :/

**Comment by Crow-EH** [comment]
> I have no idea why the CI fail... I was build and test the php image in local successfully. :/

There was an issue on IBM's domain, but it should be ok now. I relaunched the jobs.

**Comment by c-Rolland** [comment]
@Crow-EH ok thanks

**Comment by c-Rolland** [comment]
@mremi its ok for u? ^^

**Comment by mremi** [comment]
Yep 👍

**Pull request merged**